core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -351,7 +384,10 @@ class RealityModelTileLoader extends RealityTileLoader {
     super();
     this.tree = tree;
     this._batchedIdMap = batchedIdMap;
-    this._viewFlagOverrides = createDefaultViewFlagOverrides({ lighting: true });
+    let clipVolume;

```

Comment:
@RBBentley @pmconne does this change look correct? There were conflicts with what was merged to master due to @pmconne 's recent change to the override flags.

Reply:
Yes - net change should just be the addition of `clipVolume` when calling `createDefaultViewFlagOverrides`.